cloudinit/config/cc_growpart.py

```diff
@@ -316,6 +369,23 @@ def resize_devices(resizer, devices):
                          "failed to resize: disk=%s, ptnum=%s: %s" %
                          (disk, ptnum, e),))
 
+        if is_lvm and isinstance(resizer, ResizeGrowPart):
+            try:
+                if len(devices) == 1:
+                    (_out, _err) = subp.subp(["lvm", "lvextend", "--extents=",
+                                             "100%FREE", blockdev],
+                                             update_env={'LANG': 'C'})
+                    info.append((devent, RESIZE.CHANGED,
+                                 "Logical Volume %s extended" % devices[0],))
+                else:
+                    raise ValueError("Exactly one device should be configured "
+                                     "to be resized when using LVM. More than "
+                                     "one configured: %s" % devices)
```

Comment:
maybe just an info message.... I really think that we do not want to raise error (and have cloud-init fail) just because someone has added a pv to their root vg.

Reply:
If you util.logexc, log gets a WARN ... which really means fail in cloud-init.  I might be spewing dated knowledge, but I think if you get here, then the log will have a WARN and even if not it will have a stack trace that just doesn't give warm fuzzies. Other than this, I am happy with the PR at this point.

@blackboxsw or @OddBloke can you give your input?  in summary... we'll get here if the system has it's root filesystem on an LV that is part of a VG that includes more than one PVs.  It isn't really an error... cloud-init just doesn't know how to handle this situation.

I'm concerned that with this change, someone who has been happily using a system/image like this will now have stack traces showing up in their logs.

Reply:
I'm +1 on reducting the amount of illegible Tracebacks in cloud-init where we can. We have a couple of feature requests that are looking for easier logs to read (or less logging) from cloud-init. In this case, it oes seems that a standalone logging.info would capture the extent of the shortfall for cloud-init. It doesn't have to block this PR, but it's nice to keep in mind as "Tracebacks" in cloud-init.log do get treated as failures in integration tests and quick system inspection during triage.